packages/ckeditor5-ui/src/tooltipmanager.ts

```diff
@@ -453,3 +468,29 @@ function getTooltipData( element: HTMLElement ): TooltipData {
 		cssClass: element.dataset.ckeTooltipClass || ''
 	};
 }
+
+// Creates a simple MutationObserver instance wrapper that observes changes in the tooltip-related attributes of the given element.
+// Used instead of the MutationObserver from the engine for simplicity.
```

Comment:
```suggestion
// Creates a simple `MutationObserver` instance wrapper that observes changes in the tooltip-related attributes of the given element.
// Used instead of the `MutationObserver` from the engine for simplicity.
```